### ISSUE TITLE
Feature Request: exit early with advice when running parallel simulators on CircleCI

### ISSUE THREAD

**Issue by lyndsey-ferguson** [lyndsey-ferguson/fastlane-plugin-test_center#183]
#### Motivation Behind Feature

Circle CI does not allow `multi_scan` to create and run iOS Simulators in parallel. See this message from a Support Engineer:

> Relating to the question about multiple simulators running in a single container you are right, this is a limitation in the way our infrastructure is set up to allocate the resources that one average job needs. The reason why is because splitting those resources across multiple jobs running in a single instance would result in far fewer resources being available per job, which could cause slowness if not outright failures best practice is instead to spin up multiple separate instances that run simultaneously.

#### Feature Description

This would first require some investigation; Does Circle CI prevent any job from creating and launching multiple iOS Simulators in parallel? Or is this just a particular configuration that @jordan-romeroporter-earnin is running?

Once we find out that, we either detect for Circle CI, or detect for the inability to create and launch multiple iOS Simulators in parallel. Once we detect that we cannot proceed with the parallel simulators option, we abort the run and print to the console a method on how to set up parallel CI jobs that test batches of tests so that the same effect can be realized.

#### Alternatives or Workarounds

None, or just the early abort.

**Comment by lyndsey-ferguson** [comment]
@jordan-romeroporter-earnin can you let me know if Circle CI prevents any job from creating and launching multiple iOS Simulators in parallel? Or is this just a particular configuration that you are using?

**Comment by jordan-romeroporter-earnin** [comment]
I can only reiterate what I was told by the specialist that I'm in contact with. So that's all I can work with - based off my experience it seems to be true. My configuration is pretty trivial and does not include anything obvious that would prevent multiple iOS simulators in parallel.

**Comment by lyndsey-ferguson** [comment]
@jordan-romeroporter-earnin I don't suppose you could test this out to make sure you get the warning when on Circle CI and parallelizing simulator runs?

If so, can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-183-advise-no-parallelism-for-cirlce-ci"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.10.0](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.10.0)
